### PR TITLE
naga silk dress drop added to naga.as

### PR DESCRIPTION
**Pull request by melch-i**
idk why it says 10 files are changed it should have been 3, I think I might need some help

### CODE REVIEW

**Comment by brrritssocold** [comment]
This PR has been fixed with PR melch-i/Corruption-of-Champions-Mod#1

**Comment by brrritssocold** [comment]
Why was this PR closed? One of the CI builds (release) is a bit flaky, still trying to find out what is causing it (see PR #1322).

**Comment by brrritssocold** [comment]
@Kitteh6660 Was this PR closed because of the build failure? If so can it be re-opened?

**Comment by Stadler76** [comment]
Just reran the failing build. All green now.

File: classes/classes/Appearance.as [hunk 1]
```diff
@@ -1871,6 +1877,8 @@ package classes {
 					return "gooey";
 
 				case Skin.PLAIN:
+				case Skin.BARK:
+					return "bark";
```
Comment (Stadler76) [inline classes/classes/Appearance.as]: shouldn't that `return "barked";`?

File: classes/classes/Player.as [hunk 2]
```diff
@@ -1856,6 +1860,25 @@
 		//------------
 		// Mod-Added
 		//------------
+		
+	    //dryad score
+		public function DryadScore():Number
```
Comment (Stadler76) [inline classes/classes/Player.as]: You should refactor that to be named `dryadScore`

File: classes/classes/Scenes/Areas/Forest.as [hunk 3]
```diff
@@ -116,7 +116,7 @@ package classes.Scenes.Areas {
 			return _forestEncounter;
 		}
 		//Oh noes, tentacles!
-		public function tentacleBeastEncounterFn():void {
+		public function tentacleBeastEncounterFn():void { //Oh noes, tentacles!
 			clearOutput();
 			if (player.hasKeyItem("Dangerous Plants") >= 0 && player.inte / 2 > rand(50)) { //tentacle avoidance chance due to dangerous plants
 //				trace("TENTACLE'S AVOIDED DUE TO BOOK!");
```
Comment (Stadler76) [inline classes/classes/Scenes/Areas/Forest.as]: I guess the changes here could be safely reverted

File: classes/classes/Appearance.as (hunk 1 above)
Comment (melch-i) [reply classes/classes/Appearance.as]: idk, I think having "bark skin" sounds right.   I guess barked would work too but it sorta sounds weird.

**Pull request merged**